### ISSUE TITLE
Screenshot Quality argument not working?

### ISSUE THREAD

**Issue by alenpaulvarghese** [pyppeteer/pyppeteer#121]
The quality argument in coroutine screenshot is not working well 

The code i tested with 
```
 browser = await launch(
            headless=True,
            executablePath=EXEC_PATH
            )
page = await browser.newPage()
await page.goto(link)
await page.screenshot({'fullPage': True, 'path': './FILES/584512526/3726/Python Dictionarieswebshotbot.jpeg', 'type': 'jpeg', 'quality': 1})
```

Unfortunately the output quality is same if i change the quality argv to 100
Below is an example of a screenshot taken with quality 1

![alenpaul2001 (AlenPaulVarghese) · GitHub-webshotbot (6)](https://user-images.githubusercontent.com/38587581/83202629-9b731200-a165-11ea-928c-8fb52636b53f.JPEG)

**Is there something wrong with my code**

**Comment by Mattwmaster58** [comment]
> Is there something wrong with my code

Nope, after checking the code it looks like the quality is never sent as a parameter. However, checking the `pup2.1.1` branch I can see that it is correctly implemented there as part of the huge overhaul currently being undertaken to update the feature set of this package.

**Comment by Mattwmaster58** [comment]
If you want things to work now though, you could try installing from `pup2.1.1`: `pip install git+https://github.com/pyppeteer/pyppeteer --force-reinstall`. Only python-kwargs are supported though.

**Comment by Mattwmaster58** [comment]
If you want things to work now though, you could try installing from `pup2.1.1`: ~~`pip install git+https://github.com/pyppeteer/pyppeteer --force-reinstall`~~

**Comment by alenpaulvarghese** [comment]
I will try and report back

**Comment by alenpaulvarghese** [comment]
uninstalled old pyppeteer
`pip3 uninstall pyppeteer`
after that removed `__pycache__/` folder 
then reinstalled using the following command
`pip3 install git+https://github.com/pyppeteer/pyppeteer --user`
```
>>> print(pyppeteer.version)  
0.0.25
```
`await page.screenshot(path="newfile.jpeg", quality=1, type='jpeg')
`
returns the same file as before

pardon me if i done something wrong  i am currently just a beginner

**Comment by alenpaulvarghese** [comment]
ok so this was my mistake i forgot to mention the branch

**Comment by Mattwmaster58** [comment]
@alenpaul2001 Sorry, I gave you the wrong command. Here you go: `pip install git+https://github.com/pyppeteer/pyppeteer@pup2.1.1 --force-reinstall`

**Comment by alenpaulvarghese** [comment]
@Mattwmaster58 yeah i tried it, but its too noisy for the terminal, i planning to deploy this in a web server so it would actually be a mess.Hoping this issue to be fixed in the main branch.
Thanks for your time
